modules/selenium/src/main/java/org/testcontainers/containers/BrowserWebDriverContainer.java

```diff
@@ -143,26 +146,18 @@ public int getPort() {
     }
 
     @Override
-    protected void waitUntilContainerStarted() {
-        // Repeatedly try and open a webdriver session
-
-        AtomicInteger backoff = new AtomicInteger(1000);
-
-        driver = Unreliables.retryUntilSuccess(120, TimeUnit.SECONDS, () -> {
-            Uninterruptibles.sleepUninterruptibly(backoff.getAndUpdate(current -> (int)(current * 1.5)), TimeUnit.MILLISECONDS);
-            RemoteWebDriver driver = new RemoteWebDriver(getSeleniumAddress(), desiredCapabilities);
-            driver.getCurrentUrl();
-
-            logger().info("Obtained a connection to container ({})", BrowserWebDriverContainer.this.getSeleniumAddress());
-            return driver;
-        });
-
+    protected void containerIsStarted(InspectContainerResponse containerInfo) {
         if (recordingMode != VncRecordingMode.SKIP) {
             LOGGER.debug("Starting VNC recording");
-            VncRecordingSidekickContainer recordingSidekickContainer = new VncRecordingSidekickContainer<>(this);
+
+            // Use multiple startup attempts due to race condition between Selenium being available and VNC being available
+            VncRecordingSidekickContainer recordingSidekickContainer = new VncRecordingSidekickContainer<>(this)
+                    .withStartupAttempts(3);
```

Comment:
N.B. this has become necessary because we're now much quicker at starting the WebDriver connection; it exposes a race condition against the VNC socket. 

In my testing this led to ~5% of tests randomly failing due to the vnc recorder getting 'Connection Refused'.

Adding multiple startup attempts for the VNC recorder resolves the issue; we could potentially include a wait in the VNC recorder (or here), but that adds a bit more complexity.